### PR TITLE
NuoDB Collector readme page updates and enhancements

### PR DESCRIPTION
**Pull request by jleslie85**
Minior documentation updates.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,14 +25,18 @@ Built into this container are 4 input plugins to collect metrics from the NuoDB
 4.  `threads` - extends the [Telegraf ProcStat Input plugin](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/procstat) with per-thread data.
 Collects host machine resource consumption statistics on a regular 10s interval.
 
+To setup NuoDB Insights visual monitoring which uses the NuoDB Collector, follow the instructions on the [NuoDB Insights](https://github.com/nuodb/nuodb-insights) github page.
+
+to setup NuoDB perfomrance metric collection using NuoDB Collector when not using NuoDB Insights, follow the instruction on this page
```
Comment (vegichan) [inline README.md:30]: typo in performance. I also think that you need a capital T and a dot at the end.

File: README.md [hunk 2]
```diff
@@ -158,7 +160,7 @@ Repeat the steps above for all running NuoDB engine containers you want to monit
 
 ## Deploying NuoDB Collector using NuoDB Helm Charts
 
-NuoDB Collector will be deployed in sidecar containers using NuoDB Helm Charts.
+The NuoDB Collector is deployed in sidecar containers when deploying with the NuoDB Helm Charts.
```
Comment (vegichan) [inline README.md:163]: I dont think that this sentence is required at all here.

File: README.md [hunk 3]
```diff
@@ -184,11 +186,11 @@ nuocollector:
         data_format = "influx"
 ```
 
-# Setup on Bare Metal
+# Setup on Bare Metal Linux
 
 ## Installation
 
-These steps are for RedHat or CentOS. For other platforms, see [Telegraf Documentation](https://portal.influxdata.com/downloads/).
+These steps are for Red Hat or CentOS bare-metal hosts or VMs. For other platforms, see [Telegraf Documentation](https://portal.data.com/downloads/).
```
Comment (vegichan) [inline README.md:193]: the change to the link is incorrect.

**Review by vegichan — CHANGES_REQUESTED** [review]
some changes are required

File: README.md (hunk 1 above)
Comment (jleslie85) [reply README.md]: Thanks for finding those.

File: README.md (hunk 2 above)
Comment (jleslie85) [reply README.md]: Thanks, all the better. I was trying to rework this existing line assuming it was a desired detail to share, but I agree, seems like extra info not needed (keep it concise). 
I removed the line.

File: README.md (hunk 3 above)
Comment (jleslie85) [reply README.md]: Thanks, not even sure how I managed to change that, was only attempting to change the text portion! Fixed.

**Review by vegichan — APPROVED** [review]
👍

**Pull request merged**